### PR TITLE
Add a failing test for the URL rule

### PR DESCRIPTION
**Pull request by Kolyunya**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | none

Confirms #13656.

### CODE REVIEW

File: tests/framework/web/UrlRuleTest.php [hunk 1]
```diff
@@ -513,6 +513,20 @@ protected function getTestsForCreateUrl()
...
             [
+                'single optional param with suffix',
+                [
+                    'pattern' => '<page>',
+                    'route' => 'page/view',
+                    'defaults' => ['page' => 'index'],
+                    'suffix' => '/',
+                ],
+                [
+                    ['page/view', [], '/'],
```
Comment (SilverFire) [inline tests/framework/web/UrlRuleTest.php]: This assertion is incorrect. Only parameters with empty default values may be omitted.

File: tests/framework/web/UrlRuleTest.php [hunk 2]
```diff
@@ -513,6 +513,20 @@ protected function getTestsForCreateUrl()
...
+                    ['page/view', ['page' => 'index'], '/'],
```
Comment (SilverFire) [inline tests/framework/web/UrlRuleTest.php]: This one is incorrect as well. Suffix may be applied only to non-empty URL

File: tests/framework/web/UrlRuleTest.php [hunk 3]
```diff
@@ -513,6 +513,20 @@ protected function getTestsForCreateUrl()
...
+                    ['page/view', ['page' => 'news'], 'news/'],
```
Comment (SilverFire) [inline tests/framework/web/UrlRuleTest.php]: This case will be covered by https://github.com/yiisoft/yii2/pull/13624

**Review by SilverFire — CHANGES_REQUESTED** [review]

File: tests/framework/web/UrlRuleTest.php (hunk 2 above)
Comment (Kolyunya) [inline tests/framework/web/UrlRuleTest.php]: @SilverFire so the expected result is `//`, right?

Comment (SilverFire) [inline tests/framework/web/UrlRuleTest.php]: The correct one is empty string.
After #13624 merge the result will be correct
See https://github.com/yiisoft/yii2/pull/13624/commits/7206b2ba8a9e2a7648fa5507ce338fc851bc0a6e#diff-96fb0ef29c09f20871090c41bf5b3839R498

**Pull request closed without merging**